File: CosmicHammer Tests/HSdistributednotifications.m
//
//  HSdistributednotifications.m
//  Cosmic Hammer
//
//  Created by Chris Jones on 24/04/2016.
//  Copyright © 2016 Cosmic Hammer. All rights reserved.
//

#import "HSTestCase.h"

@interface HSdistributednotifications : HSTestCase

@end

@implementation HSdistributednotifications

- (void)setUp {
    [super setUpWithRequire:@"test_distributednotifications"];
    // Put setup code here. This method is called before the invocation of each test method in the class.
}

- (void)tearDown {
    // Put teardown code here. This method is called after the invocation of each test method in the class.
    [super tearDown];
}

- (void)testdistributednotifications {
    [self luaTestWithCheckAndTimeOut:5 setupCode:@"testDistributedNotifications()" checkCode:@"testDistNotValueCheck()"];
}

@end

File: Hammerspoon Tests/HSdistributednotifications.swift
import XCTest

@objcMembers
class HSdistributednotifications: HSTestCase {
    override func setUp() {
        super.setUpWithRequire("test_distributednotifications")
    }

    override func tearDown() {
        super.tearDown()
    }

    func testdistributednotifications() {
        luaTestWithCheckAndTimeOut(5, setupCode: "testDistributedNotifications()", checkCode: "testDistNotValueCheck()")
    }
}
